### DISCUSSION TITLE
can't send multiple embeds

### DISCUSSION THREAD

**Discussion by aurkaxi** [Rapptz/discord.py#8416] [Q&A] [answered]
as the documentation says i can send 10 embed at a single time
i made like this
```
embed1 = discord.Embed(...)
embed2 = discord.Embed(...)
embed2 = discord.Embed(...)
....9 of them
await ctx.send(embeds=(embed1, embed2, embed3,....)
```
but only first to embeds are sent. if i rewrite
`await ctx.send(embeds=(embed1, embed3, embed2,....) `
only embed1 and embed3 are sent. so only first 2 embeds are sent.
then i break them like this
```
await ctx.send(embeds=(embed1, embed2))
await ctx.send(embeds=(embed3, embed4))
```
now some embeds like embed4, embed6, embed8 didn't sent.
i am trying to workaround by 9 await to with single embed which is obviously not a good thing i guess.
and in another file i had only 2 embed listed them, and 2nd one wasn't sent. it previously did when 1st embed didn't have a url parameter.

**Comment by Rapptz** [comment]
I suspect you're hitting a bug or quirk in the client behaviour where multiple embeds with the same URL are concatenated together. This is to facilitate the 4-image embeds by Twitter. However, there isn't much information here to really pinpoint a problem. The library sends the list as-is so any oddities are probably with the client, you can verify that the bot is actually sending all the embeds by calling `fetch_message` on the message and seeing what the API returns for `len(message.embeds)`.

**Reply by aurkaxi** [reply]
kindly help me with a sample snippet.

**Reply by tookender** [reply]
> kindly help me with a sample snippet.

You heard Danny, use fetch_message on the message and see what len(message.embeds) returns.

**Comment by Hypurrnating** [comment]
that doesnt look like a list?

**Reply by aurkaxi** [reply]
Hi, forgot about this post. Ir happened because i have same url for all.

**Comment by aurkaxi** [comment, marked as answer]
Apparently you can't have same url for different embeds.

**Reply by LeoCx1000** [reply]
Yes. This is a special behaviour discord has (I believe it is undocumented) where the images of the embeds with the same URL (up to 4) will get grouped and tiled in the first embed (the info of the other 3 embeds won't get displayed except for the image)

**Reply by aisflat439** [reply]
Thanks for this response. Kinda weird behavior but man it bit me in the butt this morning!
